docrane/container.py

```diff
@@ -65,6 +73,19 @@ def ensure_running(self, force_restart=False):
                     LOG.warning("Image ID has changed. Restarting container.")
                 break
 
+        try:
+            image_found = util.pull_image(
```

Comment:
sounds like this method should raise an exception in the utility, rather than a generic assertion thrown. 

or, just check the return status of the image, and return if it's False.